lib/osf-components/addon/components/file-metadata-manager/component.ts

```diff
@@ -63,31 +70,37 @@ export default class FileMetadataManagerComponent extends Component<Args> {
     constructor(owner: unknown, args: Args) {
         super(owner, args);
         assert(
-            'You will need pass in a file object to the FileMetadataManager component to get metadata',
-            Boolean(args.file),
+            'You will need pass in a FileModel object to the FileMetadataManager component to get metadata',
+            Boolean(args.file instanceof FileModel),
         );
         try {
             taskFor(this.getGuidMetadata).perform();
-            this.target = this.file.target as AbstractNodeModel;
-            this.userCanEdit = this.target.currentUserPermissions.includes(Permission.Write);
-            taskFor(this.getTargetMetadata).perform();
-            this.changeset = buildChangeset(this.metadata, null);
+            taskFor(this.getTarget).perform();
         } catch (e) {
             const errorTitle = this.intl.t('osf-components.file-metadata-manager.error-getting-metadata');
             this.toast.error(getApiErrorMessage(e), errorTitle);
         }
     }
 
+    @task
+    @waitFor
+    async getTarget() {
+        this.target = await this.file.target as NodeModel;
+        this.targetLicense = await this.target.license;
+        this.targetInstitutions = await this.target.affiliatedInstitutions as InstitutionModel[];
+        this.userCanEdit = this.target.currentUserPermissions.includes(Permission.Write);
+        await taskFor(this.getTargetMetadata).perform();
+    }
+
     @task
     @waitFor
     async getGuidMetadata() {
-        if (this.file) {
-            const guidRecord = await this.store.findRecord('guid', this.file.guid, {
-                include: 'metadata',
-                resolve: false,
-            });
-            this.metadata = guidRecord.customMetadata as CustomFileMetadataRecordModel;
-        }
+        const guidRecord = await this.store.findRecord('guid', this.file.guid, {
+            include: 'metadata',
```

Comment:
should this be `include: 'custom_metadata'` ?

Reply:
Prooobably